src/views/Events/index.jsx

```diff
@@ -109,6 +109,32 @@ const Events = () => {
     setURLParams(!includePast, filters);
   };
 
+  function searchBarLength() {
+    if (width >= 1800 && width <= 1920) {
+      return 7.19;
+    } else if (width >= 1700 && width < 1800) {
+      return 6.89;
+    } else if (width >= 1600 && width < 1700) {
+      return 6.55;
+    } else if (width >= 1500 && width < 1600) {
+      return 6.31;
+    } else if (width >= 1400 && width < 1500) {
+      return 7.15;
+    } else if (width >= 1300 && width < 1400) {
+      return 6.76;
+    } else if (width >= 1200 && width < 1300) {
+      return 6.3;
+    } else if (width >= 1100 && width < 1200) {
+      return 6.93;
+    } else if (width >= 1000 && width < 1100) {
+      return 6.39;
+    } else if (width >= 920 && width < 1000) {
+      return 5.87;
+    } else {
+      return 5.87;
+    }
+  }
```

Comment:
Can you explain the goal of this function? It looks like you're trying to dynamically resize the search bar, but I'm not sure what the significance of each size is - were you trying to prevent it from overflowing onto multiple lines? If so, the simpler way to do that is to pass `true` as the value of the `xs` prop for the Grid item in questions. With `xs={true}`, the Grid item will dynamically resize itself to fill the leftover space without overflowing.

Reply:
That was the idea of this-I didn't know about that, it'd be much simpler than this function. I'll look into it!!